src/testers.jl

```diff
@@ -12,14 +12,16 @@ at input point `z` to confirm that there are correct `frule` and `rrule`s provid
 - `fdm`: the finite differencing method to use.
 - `fkwargs` are passed to `f` as keyword arguments.
 - If `check_inferred=true`, then the inferrability (type-stability) of the `frule` and `rrule` are checked.
+- `testset_name`: if provided, the name of the testset used to wrap the tests.
```

Comment:
```suggestion
- `testset_name`: if provided, the name of the testset used to wrap the tests.
  Otherwise it is determined from the function and argument types.
```